pandas/tests/frame/indexing/test_indexing.py

```diff
@@ -202,6 +202,29 @@ def test_setitem_list_of_tuples(self, float_frame):
         expected = Series(tuples, index=float_frame.index, name="tuples")
         tm.assert_series_equal(result, expected)
 
+    @pytest.mark.parametrize("columns", [["A", "E"], ["E", "F"]])
+    @pytest.mark.parametrize(
+        "box",
+        [
+            lambda x: 1,
+            lambda x: [1, 2],
+            lambda x: np.array([1, 2]),
+            lambda x: x[["B", "C"]],
+            lambda x: x[["B", "A"]].values,
+            lambda x: x[["A", "C"]].values.tolist(),
+        ],
+    )
+    def test_setitem_list_missing_columns(self, float_frame, columns, box):
+        # GH 26534
+        result = float_frame.copy()
```

Comment:
I would rather you not construct the expected frame like this, rather include it as another argument in the parameterization; essentially hard code it.